modules/widgets/search.php

```diff
@@ -270,21 +275,41 @@ public function jetpack_search_populate_defaults( $instance ) {
 	public function widget( $args, $instance ) {
 		$instance = $this->jetpack_search_populate_defaults( $instance );
 
-		$display_filters = false;
-
 		if ( ( new Status() )->is_development_mode() ) {
-			echo $args['before_widget'];
+			echo $args['before_widget']; //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
is there a specific use case on why we aren't escaping the before_widget and after_widget? It is fairly common not to but ideally we should escape.

Reply:
cc @gibrown

Reply:
@mdbitz basically because that is what we seem to do elsewhere and I think it does not necessarily work otherwise. I would be fine adding some escaping around it though I guess there is a risk it will break something existing.

Reply:
Yeah in VIP it was mostly ignored as well for this same reason. `wp_kses_post` is potentially the least destructive but let's leave as is for now.